pkg/sync/filepath_sync.go

```diff
@@ -28,6 +33,7 @@ func (fs *FilePathSync) Fetch(_ context.Context) (string, error) {
 	if err != nil {
 		return "", err
 	}
+
 	return string(rawFile), nil
```

Comment:
I was checking the file extension here in [an earlier commit](https://github.com/open-feature/flagd/pull/206/commits/3aba0064b4ce431e77e39d8eebe42e3faa3bced7) to ensure only certain file extensions like `json`, `yaml` and `yml` would be allowed but on a second thought I realized there can be files without `.json`/`.yaml`/`.yml`  extension but still have valid `json` or `yaml` content (we use such a file in our tests).

If we check the file extension, we can prevent certain cases where the user passes the wrong file as config (e.g., `config.txt` instead of `config.json` or `config.yaml`) but it also means we might have to throw an error if there is no file extension (since we don't know if the file without extension has `yaml` contents, `json` contents or something else).

Reply:
I am not sure what we should do here. 
> a second thought I realized there can be files without .json/.yaml/.yml extension but still have valid json or yaml content (we use such a file in our tests).

I removed the file extension check after thinking about this ^ but if the reviewers have any other opinion/feedback I would like to hear it :pray:

Reply:
Please ignore the above comments ^ :pray: